### DISCUSSION TITLE
Change model colour through RGB.

### DISCUSSION THREAD

**Discussion by Jasonp8000** [google/model-viewer#3480] [General]
Hi everyone, 

I’m looking into changing the models colour/finish based on a RGB selector. The user can either enter in the RGB values or use the RGB chart. 

Here’s an example I came across created by Vectery : https://fb.watch/d78LGM1Dm3/

Any direction or advise would be much appreciated.

**Comment by elalish** [comment]
Did you look at our example? https://modelviewer.dev/examples/scenegraph/#changeColor

**Reply by Jasonp8000** [reply]
Hi @elalish, thank you for your reply, 

I have made some progress in changing the model color based on RGB values. However, The lighting/exposure seems too be interfering with the model color. If you have any advice to fix this issue it would be greatly appreciated. 





[rgb selector.txt](https://github.com/google/model-viewer/files/8791156/rgb.selector.txt)

**Reply by elalish** [reply]
Yes, that's intentional. It wouldn't look physically correct if the colors were not affected by the applied lighting. You can use KHR_materials_unlit extension if you want 1:1 pixel colors, but of course then it won't look physically-correct.
